exercises/leap/example_gen.go

```diff
@@ -15,7 +16,8 @@ func main() {
 		log.Fatal(err)
 	}
 	var j js
-	if err := gen.Gen("leap.json", &j, t); err != nil {
+	f := path.Join("exercises", "leap", "canonical-data.json")
```

Comment:
You probably don't have to cancel it - all the generators will still need to update to pass in only the slug rather than the filename (`leap` rather than `leap.json`), can just repurpose it to do the `Gen` update in that PR. Whichever is easier for you if you're going to work on it though.